test/verify/check-storage-lvm2

```diff
@@ -957,6 +960,25 @@ class TestStorageLvm2Destructive(storagelib.StorageCase):
         self.dialog({})
         b.wait_not_present(".pf-v6-c-alert")
 
+    @testlib.skipImage("TODO: don't know how to move the rootfs", "debian-*", "ubuntu-*", "arch")
+    # this doesn't work in ostree either, but we don't run storage tests there
+    @testlib.skipWsContainer("newly built root doesn't contain ws container")
+    @testlib.timeout(1200)
```

Comment:
That seems excessive, and just a copy pasta?

Reply:
If anything hangs there, it'll needlessly block the test and CI runner for 20 minutes.